composition/problem-set-composition.md

```diff
@@ -99,7 +99,7 @@ print(today.display_temperature())
 * title: Composition
 
 ##### !question
-Composition can be initiated by a single line of code. In this code snippet, which line is responsible for starting the relationship between `Bedroom` and `Area`?
+Composition can be initiated by a single line of code. In this code snippet, which line is responsible for starting the relationship between `Room` and `Area`?
```

Comment:
_Should_ this be `Room`? Or should it be `bedroom`? Definitely not `Bedroom`, though... 😂

Reply:
I suppose the answer is the same either way 😆 . 

@beccaelenzil Would you be the right person to ask for the original intent?

Reply:
I don't know what the original intent was, but `Room` and `Area` sounds like a good fix to me!